docs/LTS.md

```diff
@@ -35,7 +47,7 @@ Fastify 使用 GitHub Actions 来进行 CI 测试，请参阅 [GitHub 相关文
 | 系统    | YAML 工作流标签        | 包管理器                  | Node.js      |
 |---------|------------------------|---------------------------|--------------|
 | Linux   | `ubuntu-latest`        | npm                       | 10,12,14,16  |
-| Linux   | `ubuntu-16.04`         | yarn,pnpm                 | 10,12        |
+| Linux | `ubuntu-18.04`           | yarn,pnpm                 | 10,12        |
```

Comment:
not sure if this is Github display issue, but seems like the second `|` is not aligned.

Reply:
fixed.